prember-urls.js

```diff
@@ -42,12 +42,12 @@ module.exports = function () {
       urls.push(`/${p}/${uniqVersion}/${suffix}`);
     };
 
-    const oldVersions = ['1.13', '2.18', '3.28', '4.4', '4.8', '4.12'];
+    const oldVersions = ['5.4', '5.8', '5.12'];
```

Comment:
```suggestion
    const oldVersions = ['1.13', '2.18', '3.28', '4.4', '4.8', '4.12', '5.4', '5.8', '5.12'];
```

I intended it to be additive